src/reducers/usersReducer.js

```diff
@@ -0,0 +1,27 @@
+import initialState from '../store/initialState';
+import {
+  REGISTER_SUCCESS,
+  VERIFY_USER
+} from '../actions/actionTypes';
+
+const { user } = initialState;
+const userReducer = (state = user, action) => {
+  const { type } = action;
+  switch (type) {
+    case REGISTER_SUCCESS: {
+      return {
+        ...state
```

Comment:
isn't this returning the default state? If it is then you can move it to line 22.